webhooks-extension/README.md

```diff
@@ -28,7 +82,7 @@ docker build -f cmd/extension/Dockerfile_test .
 
 ### Example creating a webhook
 
-You should be able to use the extension via the Tekton Dashboard UI - however, until the UI is coded or if you would prefer to interact with REST endpoint directly, you can create your webhook using curl:
+We would recommend using the extension via the Tekton Dashboard UI - however, if you would prefer to interact with REST endpoint directly, you can create your webhook using curl:
```

Comment:
please could you fix 'if you would prefer to interact with REST endpoint directly' while you're at it?